packages/react-ui/src/themes/light.js

```diff
@@ -447,7 +447,8 @@ const components = {
   },
   FormLabel: {
     fontSize: 2,
-    marginBottom: 1
+    marginBottom: 1,
+    color: 'text.body'
```

Comment:
Changed this in theme because otherwise the component was taking color from Grid style which was outside the light ThemeProvider